### ISSUE TITLE
[Code Addition Request]: 

### ISSUE THREAD

**Issue by Kaibalya27** [UTSAVS26/PyVerse#749]
### Have you completed your first issue?

- [X] I have completed my first issue

### Guidelines

- [X] I have read the guidelines
- [ ] I have the link to my latest merged PR

### Latest Merged PR Link

N/A

### Project Description

# Project Description

 The Crater Detection Model is an advanced object detection system developed using YOLOv8, aimed at identifying all types of lunar craters on the moon's surface. This model is particularly effective in low-light conditions, making it suitable for analyzing regions such as Permanently Shadowed Regions (PSR) where visibility is limited.

# Key Features:
 - Object Detection: Accurately detects and localizes craters in lunar images.
 - Training Dataset: Utilizes the Impact Moon Craters (LU3M6TGT) dataset for robust training and validation.
 - Flexible Input: Designed to process images of various resolutions, enhancing usability across different applications.

 This model serves as a valuable tool for lunar exploration, enabling researchers and 
 enthusiasts to gain insights into the moon's geological features and inform future missions.

### Full Name

Kaibalya Mohapatra

### Participant Role

GSSOC, HACKTOBERFEST

**Comment by Kaibalya27** [comment]
Update: Cloning error due to invalid file path on Windows

I attempted to clone the repository, but I encountered an error related to an invalid file path on my Windows system. The error seems to be caused by a folder name ending with a trailing period, which is not allowed in Windows.

Here is the error I received:
```
error: invalid path 'Machine_Learning/Linear regression using gradient decent./Linear regression using gradient decent.ipynb'
fatal: unable to checkout working tree
```
This issue prevents me from successfully cloning the repo and setting up my environment.
